tus-clients/java/src/main/java/cdc/gov/upload/client/FileUploader.java

```diff
@@ -1,22 +1,27 @@
 package cdc.gov.upload.client;
 
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
 import java.net.URL;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Properties;
-import java.util.UUID;
+import java.util.*;
 
-import cdc.gov.upload.client.model.FileStatus;
+import cdc.gov.upload.client.model.*;
 import cdc.gov.upload.client.tus.TusUploadExecutor;
 import cdc.gov.upload.client.utils.LoginUtil;
+import cdc.gov.upload.client.utils.ReadJsonFromAzureStorage;
 import cdc.gov.upload.client.utils.StatusUtil;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 public class FileUploader {
 
     public static void main(String[] args) {
```

Comment:
We need a path here to choose from running 1 destination or all the destinations.    Smoke test vs regression.

Reply:
Done.